proxy.c

```diff
@@ -69,6 +72,7 @@ void forward_data_ext(int source_sock, int destination_sock, char *cmd);
 int parse_options(int argc, char *argv[]);
 
 int server_sock, client_sock, remote_sock, remote_port = 0;
+int active_connections = 0;
```

Comment:
I'd call this variable connections_processed to reflect what's returned in status. For active_connections, the number would have to be decreased on disconnect.